common/src/main/java/org/dash/wallet/common/services/TransactionMetadataProvider.kt

```diff
@@ -47,30 +47,55 @@ interface TransactionMetadataProvider {
     suspend fun getAllTransactionMetadata(): List<TransactionMetadata>
 
     // Address methods
+    /**
+     * mark an address with a tax category.  This will replace existing data
+     *
+     * @param address the address to mark
+     * @param isInput the address is an input in a transaction
+     * @param taxCategory the tax category
+     * @param service the name of the service associated with this address
+     */
     suspend fun markAddressWithTaxCategory(
         address: String,
-        sendTo: Boolean,
+        isInput: Boolean,
```

Comment:
this was using the old `sendTo` nomenclature, but really it should be `isInput` meaning: is the address used in an input to a transaction.